dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, *args, **kwargs):
         self._placebo_type = kwargs.pop("placebo_type",None)
         if self._placebo_type is None:
             self._placebo_type = "Random Data"
-        self._num_simulations = kwargs.pop("num_simulations",200)
+        self._num_simulations = kwargs.pop("num_simulations",100)
```

Comment:
same comment. Better to have a member like self.default_num_simulations that you can use everywhere.

Reply:
Added CausalRefuter.DEFAULt_NUM_SIMULATIONS